internal/rpc/server.go

```diff
@@ -96,6 +119,9 @@ func (s *server) Listen() error {
 	s.router.HandleFunc("/rpc/v{version}/{type}/{name}", s.rpcHandler)
 	s.router.HandleFunc("/rpc/v{version}/{type}", s.rpcHandler)
 
+	s.router.Use(prometheusMiddleware)
+	s.router.Path("/rpc/metrics").Handler(promhttp.Handler())
```

Comment:
Should this be `/metrics`, it really isn't part of the RPC and perhaps the user want to proxy just `/rpc` to goaurrpc and keep `/metrics` "hidden".
```suggestion
	s.router.Path("/metrics").Handler(promhttp.Handler())
```

Reply:
Hmm we already have metrics for the aurweb from gunicorn at `/metrics` endpoint. Maybe expose them as `/rpc-metrics`?

Reply:
`goaurrpc` is a standalone application, so IMO it should just use the default path (`/metrics`). If we need it remapped in arch's infrastructure we can do it with Nginx.

Reply:
good point!